src/core/Microsoft.PowerToys.Settings.UI.Lib/Utilities/Helper.cs

```diff
@@ -41,6 +45,29 @@ public static string GetSerializedCustomAction(string moduleName, string actionN
             return sendCustomAction.ToJsonString();
         }
 
+        public static FileSystemWatcher GetFileWatcher(string moduleName, string fileName, Action onChangedCallback)
+        {
+            var watcher = new FileSystemWatcher();

```

Comment:
I'm also using it for the first time yes it will trigger an event on change made in the file based on the NotifyFilter used.  Afaik there is no way currently with `FileSystemWatcher` to know if the events is triggered at the completion of write operation and it also raise multiple events.